### PR TITLE
[New Rule] Chkconfig Service Add

### PR DESCRIPTION
**Pull request by DefSecSentinel**
## Issues

https://github.com/elastic/detection-rules/issues/2153

## Summary

Detects the use of the chkconfig binary to manually add a service for management by chkconfig. Threat actors may utilize this technique to maintain persistence on a system. When a new service is added, chkconfig ensures that the service has either a start or a kill entry in every runlevel and when the system is rebooted the service file added will run providing long-term persistence.

### CODE REVIEW

File: rules/linux/persistence_chkconfig_service_add.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+license = "Elastic License v2"
+name = "Chkconfig Service Add"
+references = [
+    "https://www.intezer.com/blog/research/lightning-framework-new-linux-threat/"
+]
+risk_score = 74
+rule_id = "b910f25a-2d44-47f2-a873-aabdc0d355e6"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Persistence", "Lightning Framework"]
+type = "eql"
```
Comment (DefSecSentinel) [inline rules/linux/persistence_chkconfig_service_add.toml:23]:
```suggestion
timestamp_override = "event.ingested"
type = "eql"
```

**Review by imays11 — APPROVED** [review]

File: rules/linux/persistence_chkconfig_service_add.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2022/07/22"
+maturity = "production"
+updated_date = "2022/07/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the use of the chkconfig binary to manually add a service for management by chkconfig. Threat actors may utilize this technique to maintain persistence on a system. When a new service is added, chkconfig ensures that the service has either a start or a kill entry in every runlevel and when the system is rebooted the service file added will run providing long-term persistence.
```
Comment (terrancedejesus) [inline rules/linux/persistence_chkconfig_service_add.toml:9]: TOML lint.

**Review by terrancedejesus — APPROVED** [review]
LGTM. Recommended linting for the description.

**Review by shashank-elastic — APPROVED** [review]
Lint and we are good to merge

**Pull request merged**